packages/backend/server/src/data/migrations/1721814446774-update-prompts.ts

```diff
@@ -0,0 +1,13 @@
+import { PrismaClient } from '@prisma/client';
+
+import { refreshPrompts } from './utils/prompts';
+
+export class UpdatePrompts1721814446774 {
```

Comment:
we definitely need a way to manage prompts

Reply:
i prefer have a bootstrap in prompt module, it will refresh the prompt if not modified in runtime

Reply:
do it

Reply:
next pr